drivers/scheduler/k8s/k8s.go

```diff
@@ -1801,6 +1801,69 @@ func (k *k8s) createMigrationObjects(
 	return nil, nil
 }
 
+func (k *k8s) getScName(pvc *v1.PersistentVolumeClaim) string {
```

Comment:
the logic is incorrect here. The provisioner name is never in the pvc labels. Also this function will no longer be required after changing below function to use https://github.com/portworx/sched-ops/blob/master/k8s/k8s.go#L3023.